x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -72,6 +72,17 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'linux.advanced.artifacts.global.channel',
+    first_supported_version: '8.18',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.linux.advanced.artifacts.global.channel',
+      {
+        defaultMessage:
+          'The release channel to use for receiving global artifacts. By default, staged roll-out is applied. Set to rapid to receive candidate artifacts as soon as available. Set to stable to only receive stable artifacts. Default: default',
```

Comment:
The sentence here is missing an article. `By default, a staged roll-out is applied.` (same for the changes in other places). Also agree with adding quotes to the keywords here "staged". 

`Set this value to "stable"/Set this value to "rapid"` is maybe better to read than the terse version here.

Reply:
I suggest removing the need for setting a `default` in the input. If no value is given then it is set to default.

Reply:
I wouldn't add an 'a' article to 'staged roll-out', as it's not an item, but a process or a concept.

there's actually no need for setting a default, it's just a common scheme in the advanced options to define how an empty input is considered by default